server/model/environments.go

```diff
@@ -0,0 +1,47 @@
+package model
+
+import (
+	"strings"
+	"time"
+)
+
+type (
+	Environment struct {
+		ID          string
+		Name        string
+		Description string
+		CreatedAt   time.Time
+		Values      []EnvironmentValue
```

Comment:
shouldn't this be a `map[string]string`? maybe an `OrderedMap`?

Reply:
The order of environments doesn't matter, so no need for OrderedMap. `map[string]string` might be a better alternative than an array.